test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -7,6 +7,16 @@ class Api::V2::OrganizationsControllerTest < ActionController::TestCase
     assert_not_nil assigns(:organizations)
   end
 
+  # assign_lone_taxonomies causes internal server error in case the content is not JSON and contains
+  # organization data should it be called before the media type check.
+  # We must make sure the media type check is performed before assign_lone_taxonomies is called.
+  test "should not trigger internal server error on wrong content type" do
```

Comment:
Nit: some guides say to avoid "should not" wording and instead focus on what it does. Perhaps this is a better description?
```suggestion
  test "handles unsupported content type" do
```

Reply:
Thanks, updated.